src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateRuntimeGraph.cs

```diff
@@ -75,21 +75,38 @@ public ITaskItem SourceRuntimeJson
         /// <summary>
         /// Where to write the final runtime.json
         /// </summary>
-        [Required]
         public ITaskItem RuntimeJson
         {
             get;
             set;
         }
 
+        /// <summary>
+        /// When defined, specifies the file to write compatibility precedence for each RID in the graph.
+        /// </summary>
+        public ITaskItem CompatibilityMap
```

Comment:
Any reason why these are TaskItems instead of just `string` properties?

Reply:
Nope, I can make them strings.